src/test/java/com/bravo/user/service/PaymentServiceTest.java

```diff
@@ -0,0 +1,141 @@
+package com.bravo.user.service;
+
+import com.bravo.user.App;
+import com.bravo.user.dao.model.Payment;
+import com.bravo.user.dao.model.mapper.ResourceMapper;
+import com.bravo.user.dao.repository.PaymentRepository;
+import com.bravo.user.dao.specification.PaymentSpecification;
+import com.bravo.user.model.dto.PaymentDto;
+import com.bravo.user.model.filter.PaymentFilter;
+import com.bravo.user.utility.PageUtil;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+@ContextConfiguration(classes = {App.class})
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+public class PaymentServiceTest {
+
+    private static final String PAGE_COUNT = "page-count";
+    private static final String PAGE_NUMBER = "page-number";
+    private static final String PAGE_SIZE = "page-size";
+    @Autowired
+    private HttpServletResponse httpResponse;
+    @Autowired
+    private PaymentService paymentService;
+    @MockBean
+    private ResourceMapper resourceMapper;
+    @MockBean
+    private PaymentRepository paymentRepository;
+
+    private List<PaymentDto> paymentDtos;
+    private PaymentFilter paymentFilter;
+
+    @BeforeEach
+    public void beforeEach() {
+        final List<Integer> ids = IntStream
+                .range(1,6)
+                .boxed()
+                .collect(Collectors.toList());
```

Comment:
I keep forgetting List.of is an item of Java9 and above. I am used to mainly working in Java8. I will look into it and see which performs better.

Reply:
It looks like in this scenario, List.of(), would be better; given that the amount of data is smaller and has no reason to be changed (since it is immutable).